login.html

```diff
@@ -129,6 +129,8 @@
     var data = new FormData();
     data.append('email', email.value);
     data.append('password', password.value);
+    email.value = "";
```

Comment:
I think you should only do this when you get an error back from registration. We shouldn't do this on login failure, for example.